### PR TITLE
Add unit tests for booking

### PR DESCRIPTION
**Pull request by nerrons**

### CODE REVIEW

**Comment by sreycodes** [comment]
> Fuck Codacy cuz it makes no sense

Lol just remove the unused variables and ```.equals(null)``` to ```== null```. 
Also you might wanna refrain from such language as the module is staff monitored.

File: src/main/java/seedu/hms/model/booking/Booking.java [hunk 1]
```diff
@@ -69,10 +69,36 @@ public int numOfCustomers() {
...
+        if (other == this) {
+            return true;
+        }
+
+        if (!(other instanceof Booking)) {
+            return false;
+        }
+
+        Booking ob = (Booking) other;
+        System.out.println(getOtherUsers());
```
Comment (sreycodes) [inline src/main/java/seedu/hms/model/booking/Booking.java]: Can remove print statements ?

File: src/main/java/seedu/hms/model/booking/Booking.java [hunk 2]
```diff
@@ -69,10 +69,36 @@ public int numOfCustomers() {
...
+        System.out.println(ob.getOtherUsers());
+        return ob.getService().equals(getService())
+                && ob.getTiming().equals(getTiming())
+                && ob.getPayer().equals(getPayer())
+                && ob.getOtherUsers().equals(getOtherUsers());
```
Comment (sreycodes) [inline src/main/java/seedu/hms/model/booking/Booking.java:100]: Does this check the contents of the optional lists?

File: src/test/java/seedu/hms/logic/commands/AddBookingCommandIntegrationTest.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    @Test
+    public void executeNewBookingSuccess() {
+        Booking validBooking = new BookingBuilder()
+                .withService(ServiceType.GAMES)
+                .withTiming(14, 15)
+                .withPayer(TypicalCustomers.BOB)
+                .withOtherUsers()
+                .withComment("CoolComment")
+                .build();
+        System.out.println(validBooking);
```
Comment (sreycodes) [inline src/test/java/seedu/hms/logic/commands/AddBookingCommandIntegrationTest.java]: Remove print statements

**Review by sreycodes — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/hms/model/booking/Booking.java (hunk 1 above)
Comment (sreycodes) [reply src/main/java/seedu/hms/model/booking/Booking.java]: Didn't check all files - but just remove from all

File: src/main/java/seedu/hms/model/booking/Booking.java (hunk 2 above)
Comment (nerrons) [reply src/main/java/seedu/hms/model/booking/Booking.java:100]: Yes, it's automatically handled

File: src/main/java/seedu/hms/model/booking/Booking.java (hunk 1 above)
Comment (nerrons) [reply src/main/java/seedu/hms/model/booking/Booking.java]: I removed all in the new commits, thanks for the review

**Review by sreycodes — APPROVED** [review]

**Pull request merged**